utils_cv/multi_object_tracking/convert_seq_vid.py

```diff
@@ -0,0 +1,83 @@
+import cv2
```

Comment:
Please don't put helper functions into their own files. Should be similar to other scenarios where we have data.py, dataset.py, model.py, etc.